### PR TITLE
add navixy theme design

### PR DESCRIPTION
**Pull request by z808z**

### CODE REVIEW

File: theme/partials/search.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="md-search" data-md-component="search" role="dialog">
```
Comment (fenik17) [inline theme/partials/search.html:1]: Missed closed tag for the first `div`

File: theme/assets/stylesheets/extra.css [hunk 2]
```diff
@@ -1,37 +1,463 @@
...
+}
+
+.md-sidebar {
+    width: 11rem;
+    top: 2.4rem !important;
+    padding: 0;
+}
+
+.md-sidebar.md-sidebar--secondary {
+    direction: rtl;
```
Comment (fenik17) [inline theme/assets/stylesheets/extra.css:322]: Why RTL? This looks strange:
изображение

File: theme/partials/search.html (hunk 1 above)
Comment (z808z) [reply theme/partials/search.html:1]: fixed it

File: theme/assets/stylesheets/extra.css (hunk 2 above)
Comment (z808z) [reply theme/assets/stylesheets/extra.css:322]: RTL is made for scrollbar on the left side.
Visually, everything is fine.
https://i.paste.pics/0004c0396652e9f300acfd1702a424c3.png

Comment (z808z) [reply theme/assets/stylesheets/extra.css:322]: fixed

**Pull request merged**